public/app/features/alerting/unified/RuleEditor.test.tsx

```diff
@@ -20,6 +20,8 @@ import { ExpressionEditorProps } from './components/rule-editor/ExpressionEditor
 import { GrafanaAlertStateDecision } from 'app/types/unified-alerting-dto';
 import { searchFolders } from '../../../../app/features/manage-dashboards/state/actions';
 
+jest.setTimeout(60 * 1000);
```

Comment:
Why do we need this?

Reply:
The build was failing because this test exceeded the timeout for some reason. I don't think it was related to any changes I made but figured I'd bump it just to get it passing again